### PR TITLE
Backport fixes proposed in the plugin generator for the 2.x branch

### PR DESCRIPTION
**Pull request by purbon**

### CODE REVIEW

File: lib/pluginmanager/templates/filter-plugin/lib/logstash/filters/example.rb.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@ class LogStash::Filters::<%= classify(plugin_name) %> < LogStash::Filters::Base
     if @message
       # Replace the event message with our message as configured in the
       # config file.
-      event["message"] = @message
+      event.set("message", @message)
```
Comment (wiibaa) [inline lib/pluginmanager/templates/filter-plugin/lib/logstash/filters/example.rb.erb]: @purbon I suppose you do not want this in 2.x branch

File: lib/pluginmanager/templates/filter-plugin/spec/filters/example_spec.rb.erb [hunk 2]
```diff
@@ -15,7 +15,7 @@ describe LogStash::Filters::<%= classify(plugin_name) %> do
 
     sample("message" => "some text") do
       expect(subject).to include("message")
-      expect(subject['message']).to eq('Hello World')
+      expect(subject.get('message')).to eq('Hello World')
```
Comment (wiibaa) [inline lib/pluginmanager/templates/filter-plugin/spec/filters/example_spec.rb.erb]: @purbon I suppose you do not want this in 2.x branch

Comment (purbon) [inline lib/pluginmanager/templates/filter-plugin/spec/filters/example_spec.rb.erb]: agree, good catch. Me just backported all your commits.

**Comment by suyograo** [comment]
LGTM

**Pull request merged**